docs/TelemetryDetails.md

```diff
@@ -0,0 +1,407 @@
+`customDimensions.IsAlreadyLoggedIn` | `True` if the IssueReporter has all needed config information, otherwise `False`.
+`customDimensions.IssueReporter` | The ServiceName property of the associated issue reporter. Current values are `Azure Boards` for ADO or `GitHub` for GitHub.
+
+#### TabStop_Record_On
+Trigger: The user begins to record tab stops.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.Scope` | The scope of the selection. Current values are `App` or `Element`.
+
+#### TabStop_Select_Records
+Trigger: The user selects one or more tab stops that have been recorded.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.By` | The count of selected items.
+
+#### TestSelection_Set_Scope
+Trigger: The user sets the selection scope via the command bar.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.Scope` | The scope of the selection. Current values are `App` or `Element`.
+
+#### Test_Requested
+Trigger: The user triggers an automated scan.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.By` | The trigger mechanism. Current values are `HotKey`, `Beaker`, `HierarchyMode`, or `Timer`.
+`customDimensions.Scope` | The scope of the selection. Current values are `App` or `Element`.
+
+#### Upgrade_DoInstallation
+Trigger: The user completes an install from the Upgrade dialog.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.UpdateInstallerUpdateTime` | The time it took to download the new MSI, validate it, and launch the VersionSwitcher. Does not include the execution time of the VersionSwitcher. Format is Hours:Minutes:Seconds.FractionalSeconds.
+`customDimensions.UpdateResult` | The result of the operation. Current values are `Unknown`, `Success`, or `RequiredUpgrade`.
+
+#### Upgrade_GetUpgradeOption
+Trigger: The application checks for upgrade at application startup.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.UpdateInitializationTime` | The updater's measurement of how long it took to determine the UpdateOption. Format is Hours:Minutes:Seconds.FractionalSeconds.
+`customDimensions.UpdateOptionWaitTime` | The app's measurement of how long it took to determine the UpdateOption. It is different from UpdateInitializationTime, since they start at different times on different threads. Format is Hours:Minutes:Seconds.FractionalSeconds.
+`customDimensions.UpdateOption` | The UpdateOption that was returned to the app (will be ignored if we timed out).
+`customDimensions.UpdateTimedOut` | `True` if the AutoUpdate process exceeded the 2 second timeout, otherwise `False`.
+
+#### Upgrade_InstallationError
+Trigger: The user triggers an install from the Upgrade dialog, and an error occurred.  
+Additional properties: 
+Name | Value
+--- | ---
+`customDimensions.Error` | The UpdateResult, which is always `Unknown`. *Note: Should this provide more useful information?*
```

Comment:
Should we attempt to address "note:" annotations in this PR? If not now, when?

Reply:
Future PR, not sure when